gradle.properties

```diff
@@ -1 +1,3 @@
-org.gradle.jvmargs=-Xmx2048m
+# Bump this if you're encountering OutOfMemory errors,
+# though needing more might be a sign that something is wrong in the build tool.
+org.gradle.jvmargs=-Xmx3072m
```

Comment:
oh no it's time to download more ram
(although i think the 3 GBs limit should be fine on a 8 GB PC)
(you can ignore this comment)